posthog-ai/src/vercel/middleware.ts

```diff
@@ -49,14 +49,23 @@ const mapVercelParams = (params: any): Record<string, any> => {
 }
 
 const mapVercelPrompt = (prompt: LanguageModelV1Prompt): PostHogInput[] => {
-  return prompt.map((p) => {
+  // normalize single inputs into an array of messages
+  let promptsArray: any[]
+  if (typeof prompt === 'string') {
```

Comment:
it'd be cool to create a util for this eg `isString(obj)` since we repeat this often and it is very verbose